vj4/model/record.py

```diff
@@ -22,23 +22,25 @@
 STATUS_COMPILING = 21
 STATUS_IGNORED = 30
 
+TYPE_SUBMISSION = 0
```

Comment:
A judge request (record) will either be a submission or a (pre)test.
All records from old-vj are submissions.